File: isegoria/src/main/java/com/eulersbridge/isegoria/network/PollResultsResponse.java
package com.eulersbridge.isegoria.network;

import com.eulersbridge.isegoria.models.PollResult;
import com.squareup.moshi.Json;

import java.util.List;

public class PollResultsResponse {

    @Json(name = "nodeId")
    public long id;

    public long pollId;

    @Json(name = "answers")
    public List<PollResult> results;

}

File: isegoria/src/main/kotlin/com/eulersbridge/isegoria/network/api/response/PollResultsResponse.kt
package com.eulersbridge.isegoria.network.api.response

import com.eulersbridge.isegoria.network.api.model.PollResult
import com.squareup.moshi.Json

data class PollResultsResponse (
    @field:Json(name = "nodeId")
    val id: Long,

    val pollId: Long,

    @field:Json(name = "answers")
    val results: List<PollResult>
)